content/en/dashboards/widgets/_index.md

```diff
@@ -123,6 +123,19 @@ Custom links connect data values to URLs such as a Datadog page or your AWS cons
 
 To customize interactions with data inline your generic widgets, see [Custom Links][2].
 
+## Unit overrides
+
+Customize unit values displayed on widgets to add context to your data. 
+- Unit override: choose to display units in the family of 'memory', and have Datadog take care of displaying the appropriate scale depending on data (such as megabytes or gigabytes).
+- Unit and scale override: fix units to a single scale (display data in megabytes regardless of value).
+- Define custom units: define completely custom units (like 'tests' instead of a generic count).
```

Comment:
```suggestion
- **Unit override**: choose to display units in the family of 'memory', and have Datadog take care of displaying the appropriate scale depending on data (such as megabytes or gigabytes).
- **Unit and scale override**: fix units to a single scale (display data in megabytes regardless of value).
- **Define custom units**: define completely custom units (like 'tests' instead of a generic count).
```